app/assets/javascripts/ordered_creator.js

```diff
@@ -27,25 +27,34 @@ function swapDown(selector) {
 }
 
 function reindexNestedOrderedField(mutationsList) {
-  hidden_index_selectors = $(".nested-ordered-creator .index");
-  hidden_index_selectors.each(function (index) {
-    hidden_index_selectors[index].value = index;
-  });
-  hidden_index_selectors = $(".nested-ordered-title .index");
-  hidden_index_selectors.each(function (index) {
-    hidden_index_selectors[index].value = index;
-  });
 
-  hidden_index_selectors = $(".nested-ordered-related-items .index");
-  hidden_index_selectors.each(function (index) {
-    hidden_index_selectors[index].value = index;
-  });
+  nested_fields = ['.nested-ordered-creator .index', 
+                     '.nested-ordered-title .index', 
+                     '.nested-ordered-abstract .index', 
+                     '.nested-ordered-contributor .index', 
+                     '.nested-ordered-description .index', 
+                     '.nested-ordered-related-item .indexs']
```

Comment:
is the trailing s `indexs` a bug? I think the selector is supposed to be `.nested-ordered-related-items .index`?